charts/kafka-topic-replicator/container/app/kafka_topic_replicator.py

```diff
@@ -0,0 +1,87 @@
+import os
+import json
+import logging
+from time import sleep
+from datetime import datetime
+
+from kafka import KafkaProducer, KafkaConsumer
+
+log = logging.getLogger(__name__)
+
+def setupLogging():
```

Comment:
Best practice in python is to use lowercase and underscores.
```suggestion
def setup_logging():
```